noir-projects/aztec-nr/aztec/src/oracle/pxe_store.nr

```diff
@@ -0,0 +1,16 @@
+#[oracle(store)]
+unconstrained fn store_oracle<let N: u32>(key: Field, values: [Field; N]) {}
+
+/// Store an array of values in local PXE database. The data is scoped to the current contract.
+pub unconstrained fn store<let N: u32>(key: Field, values: [Field; N]) {
+    store_oracle(key, values);
+}
```

Comment:
I'd have these use the serde traits to be able to store and retrieve values of arbitrary types.

Reply:
https://github.com/AztecProtocol/aztec-packages/pull/10867/commits/6687768d03d5a813d3e59868e4d68d0840fbadac